generators/app/templates/src/services/utils.test.tsx

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable max-nested-callbacks, @typescript-eslint/no-empty-function */
+import React, { useEffect } from 'react';
+import { mount } from 'enzyme';
+import { PROBLEM_CODE } from 'apisauce';
+
+import { useRequest, useLazyRequest, Error } from '~services/utils';
+import { TestHook, testHook } from '~utils/tests';
+import { Nullable } from '~utils/types';
+
+type True = true;
+type False = false;
+
+interface FooError {
+  errorData: string;
+}
+
+type HookReturnValue = [
+  {
+    foo: string;
+  } | null,
+  boolean,
+  Nullable<Error<FooError>>,
+  (params?: any) => void
+];
+
+const failureResponse = {
+  ok: false as False,
```

Comment:
```suggestion
  ok: false as const,
```
does that work?

Reply:
`ok: false as false` works, I just tested it and I'll change it